shuttleofx_client/views.py

```diff
@@ -3,247 +3,281 @@
 import requests
 from functools import wraps
 from flask import (
-    request,
-    jsonify,
-    render_template,
-    abort,
-    Response,
-    redirect,
-    url_for,
-    session
+	request,
```

Comment:
Modified my IDE parameters to apply this convention.
Hopefully I will not have problems of this sort anymore.